lib/ui/src/modules/ui/components/layout/index.js

```diff
@@ -75,7 +75,8 @@ const fullScreenPreviewStyle = {
   border: 0,
   margin: 0,
   padding: 0,
-  overflow: 'hidden',
+  overflow: 'auto',
```

Comment:
I wonder if this change has any side effects in other browsers.

Reply:
LGTM